_events/2017-08-05.md

```diff
@@ -0,0 +1,27 @@
+---
+title: "MusED Hackathon"
+description: "Description Coming shortly"
+date: 2017-08-05 12:00pm
+address: "45 W 18th St, New York, NY 10011"
+#rsvp: "http://eventbrite.link.or.similar"
+#image: "place-this-image-file-in-assets-slash-events.jpg"
+---
+
+# Schedule
+
+- 11:30 Doors open
```

Comment:
In the spirit of the website always being in a state of presentable, how about "Announced soon" or something under Talks and Workshops?